build.gradle

```diff
@@ -387,8 +387,9 @@ def getGitCommit() {
 	def dotGit = new File("$projectDir/.git")
 	if (!dotGit.isDirectory()) return 'non-git build'
 
+	def projectDir = dotGit.getParentFile()
```

Comment:
I find that kind of strange, after all, we just used $projectDir a few lines above. I wonder if there is a reason why the original author did this. Maybe

```
def proc = cmd.execute(null, "$projectDir")
```

does not work?

Reply:
Please change this to  follow https://github.com/igniterealtime/Smack/commit/45e865757a7242ea4942855213b53a9670c06f9c

Reply:
Sure

Reply:
I still don't see a

```
def projectDirFile = new File("$projectDir")
```

Reply:
Ah that's what you meant. I thought you were referring to the variable name alone.